packages/snaps-controllers/src/interface/SnapInterfaceController.ts

```diff
@@ -232,6 +250,22 @@ export class SnapInterfaceController extends BaseController<
     });
   }
 
+  /**
+   * Resolve the promise of a given interface approval request.
+   *
+   * @param snapId - The snap id.
+   * @param id - The interface id.
+   * @param value - The value to resolve the promise with.
+   */
+  async resolveInterface(snapId: SnapId, id: string, value: Json) {
+    this.#validateArgs(snapId, id);
+    this.#validateApproval(id);
+
+    await this.#acceptApprovalRequest(id, value);
```

Comment:
Can we document that this is under the assumption that the interface ID matches the approval ID?

Reply:
Added some explanation in the JSDoc of this method